server/main.lua

```diff
@@ -97,14 +136,15 @@ local function createPlayerVehicle(request)
     props.fuelLevel = props.fuelLevel or 100
     props.model = joaat(request.model)
 
-    return MySQL.insert.await('INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, state) VALUES ((SELECT license FROM players WHERE citizenid = ?),?,?,?,?,?,?)', {
+    return MySQL.insert.await('INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, state, garage) VALUES ((SELECT license FROM players WHERE citizenid = ?),?,?,?,?,?,?,?)', {
         request.citizenid,
         request.citizenid,
         request.model,
         props.model,
         json.encode(props),
         props.plate,
-        State.OUT
+        request.garage and State.GARAGED or State.OUT,
```

Comment:
I noticed in the garages that the impounded vehicles are still visible in the garages, only they cannot be moved out.
Is there any connection?

Reply:
I'm not sure I understand the question

Reply:
Can you explain where this parameter is used?

Reply:
It's used in garages